torch/distributions.py

```diff
@@ -133,13 +148,23 @@ class Normal(Distribution):
         mean (float or Tensor or Variable): mean of the distribution
         std (float or Tensor or Variable): standard deviation of the distribution
     """
+
     def __init__(self, mean, std):
         self.mean = mean
         self.std = std
 
     def sample(self):
         return torch.normal(self.mean, self.std)
 
+    def sample_n(self, n):
+        # cleanly expand float or Tensor or Variable parameters
+        def expand(v):
+            if isinstance(v, Number):
+                return torch.Tensor(n).fill_(v).unsqueeze(-1)
```

Comment:
I think this can be made more efficient by doing `torch.Tensor([v]).expand(n, 1)` i.e. creating a 1D tensor from the value and expanding it to nx1